sdk/client/src/otel/exporter.ts

```diff
@@ -17,33 +21,65 @@ type SendOnErrorCallback = Parameters<OTLPTraceExporter['send']>[2]
 export class OTLPTraceExporterBrowserWithXhrRetry extends OTLPTraceExporter {
 	private readonly xhrTraceExporter: OTLPTraceExporter
 
-	constructor(config?: ExporterConfig) {
+	constructor(config?: TraceExporterConfig) {
 		super(config)
 		this.xhrTraceExporter = new OTLPTraceExporter({
 			...(config ?? {}),
 			headers: {}, // a truthy value enables sending with XHR instead of beacon
 		})
 	}
 
-	send(
-		items: ReadableSpan[],
-		onSuccess: () => void,
-		onError: SendOnErrorCallback,
-	): void {
+	export(items: any, resultCallback: (result: ExportResult) => void) {
```

Comment:
Didn't realize the difference between `send` and `export`. Definitely makes sense to have this retry logic here instead of `send` - thanks!

Reply:
this change came from the otel client-side upgrade; i believe they changed some of the API in the latest version